db/migrate/20200221120126_add_moved_agreed_to_moves.rb

```diff
@@ -0,0 +1,6 @@
+class AddMovedAgreedToMoves < ActiveRecord::Migration[5.2]
+  def change
+    add_column :moves, :move_agreed, :boolean
+    add_column :moves, :move_agreed_by, :string
```

Comment:
null: false, default: false is always a good idea for booleans, so that find_by(move_agreed: false) works as expected.

Reply:
@starswan thanks for the review!
`default: false` almost always a good idea, 
in this case `move_agreed` is  relevant only if `move_type` is a prison_transfer.
It does not apply in other cases.